packages/integrations/src/prowlarr/prowlarr-integration.ts

```diff
@@ -0,0 +1,83 @@
+import { Integration } from "../base/integration";
+import { IntegrationTestConnectionError } from "../base/test-connection-error";
+import type { Indexer } from "../interfaces/indexer-manager/indexer";
+import { indexerResponseSchema, statusResponseSchema } from "./prowlarr-types";
+
+export class ProwlarrIntegration extends Integration {
+  public async getIndexersAsync(): Promise<Indexer[]> {
+    const apiKey = super.getSecretValue("apiKey");
+
+    const indexerResponse = await fetch(`${this.integration.url}/api/v1/indexer?apikey=${apiKey}`);
```

Comment:
I would suggest to use the `x-Api-Key` header that is supported by prowlarr:
https://prowlarr.com/docs/api/
![image](https://github.com/user-attachments/assets/58d038ff-5489-4bc3-a1ec-1e87fd0585f8)